tf_encrypted/keras/models/sequential.py

```diff
@@ -106,6 +110,33 @@ def layers(self):
       return layers[1:]
     return layers[:]
 
+  def backward(self, d_y):
+    for layer in reversed(self.layers):
+      grad_weights, d_y = layer.backward(d_y)
+      self._optimizer.apply_gradients(layer.weights, grad_weights)
+
+  def compile(self, optimizer, loss=None):
+    self._optimizer = optimizers.get(optimizer)
+    self._loss = loss
+
+  def fit_batch(self, x, y):
+    y_pred = self.call(x)
+    dy = self._loss.grad(y, y_pred)
+    self.backward(dy)
+    loss = self._loss(y, y_pred)
+    sess = KE.get_session()
+    self._current_loss = sess.run(loss.reveal())
+
+  def fit(self, x, y, epochs=1, steps_per_epoch=1):
+    for e in range(epochs):
+      print('Epoch {}/{}'.format(e + 1, epochs))
```

Comment:
use logger instead (if anything)

Reply:
Fixed -  I used logger instead. But it looks quite ugly lol. Maybe there is the option to remove the pink color at least